comtypes/test/test_portabledevice.py

```diff
@@ -0,0 +1,146 @@
+ # so we don't crash when the generated files don't exist
+from __future__ import annotations
+
+from ctypes import (POINTER, _Pointer, pointer, cast,
+                    c_uint32, c_uint16, c_int8, c_ulong, c_wchar_p)
+from typing import Optional
+import unittest as ut
+
+import comtypes
+import comtypes.client
+
+comtypes.client.GetModule("portabledeviceapi.dll")
+import comtypes.gen.PortableDeviceApiLib as port_api
+
+comtypes.client.GetModule("portabledevicetypes.dll")
+import comtypes.gen.PortableDeviceTypesLib as port_types
+
+
+def newGuid(*args: int) -> comtypes.GUID:
+    guid = comtypes.GUID()
+    guid.Data1 = c_uint32(args[0])
+    guid.Data2 = c_uint16(args[1])
+    guid.Data3 = c_uint16(args[2])
+    for i in range(8):
+        guid.Data4[i] = c_int8(args[3 + i])
+    return guid
+
+
+def PropertyKey(*args: int) -> _Pointer[port_api._tagpropertykey]:
+    assert len(args) == 12
+    assert all(isinstance(x, int) for x in args)
+    propkey = port_api._tagpropertykey()
+    propkey.fmtid = newGuid(*args[0:11])
+    propkey.pid = c_ulong(args[11])
+    return pointer(propkey)
```

Comment:
```suggestion
def PropertyKey(fmtid: comtypes.GUID, pid: int) -> "_Pointer[port_api._tagpropertykey]":
    propkey = port_api._tagpropertykey()
    propkey.fmtid = fmtid
    propkey.pid = c_ulong(pid)
    return pointer(propkey)
```

It is better to give an explicit name and type to the argument than to add assert to inspect the argument.